sources/packages/web/src/views/student/AppStudent.vue

```diff
@@ -18,25 +18,43 @@ import { StudentService } from "../../services/StudentService";
   },
 })
 export default class AppStudent extends Vue {
+  // Flag to hold whole student app children mounding
+  isAuthReady = false;
   async created() {
     const router = this.$router;
+    const route = this.$route;
+    await AppConfigService.shared.initAuthService("student");
+    this.isAuthReady = true;
     if (!AppConfigService.shared.authService?.authenticated) {
       router.push({
         name: "Login",
       });
-    } else if (await UserService.shared.checkUser()) {
-      /*After checking the user exists, information differences between BC Service card and SABC is synced
-      And Redirect to Home page (Student's Dashboard)*/
-      await StudentService.shared.synchronizeFromUserInfo();
-      router.push({
-        name: "StudentDashboard",
-      });
     } else {
-      /* User doesn't exist in SABC Database and so redirect the user to Student Profile page
+      // TODO:
+      // - Try to implement a role based processing
+      // Get path
+      const path = route.path;
+      console.log("Not ok if called before KC");
+      if (path.includes("/student")) {
```

Comment:
Since the AppStudent component is loaded only under "/student" path, is there any specific reason to check if the current path includes "/student"?

Reply:
Done